examples/sql_bank/main.go

```diff
@@ -32,23 +33,21 @@ import (
 	"github.com/cockroachdb/cockroach/util/log"
 )
 
-const (
-	maxTransfer = 999
-	numAccounts = 999
-	concurrency = 5
-	aggregate   = true
-	usePostgres = false
-)
+var maxTransfer = flag.Int("maxTransfer", 999, "Maximum amount to transfer in one transaction.")
+var numAccounts = flag.Int("numAccounts", 999, "Number of accounts.")
```

Comment:
The flags should be named with hyphens, as they are in kv_bank: `var numAccounts = flag.Int("num-accounts",...)`.